dev-tools/whitelist.sh

```diff
@@ -0,0 +1,13 @@
+uhb_whitelist -f ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE-in-testing.txt -o ${TRAVIS_BUILD_DIR}/phishing-domains-ACTIVE-in-testingTEMP.txt
```

Comment:
Let's add `-m` ([cf](https://github.com/Ultimate-Hosts-Blacklist/whitelist/blob/aa56eb5783f87a702c2dd25f61ae042d71c3f0e4/ultimate_hosts_blacklist/whitelist/__init__.py#L181-L187)) to speed up the processes.

As it is working under Travis CI, let's also add `-p 60` ([cf](https://github.com/Ultimate-Hosts-Blacklist/whitelist/blob/aa56eb5783f87a702c2dd25f61ae042d71c3f0e4/ultimate_hosts_blacklist/whitelist/__init__.py#L189-L191)).

Reply:
Also, We don't need to save at a backup place as the tool only write the destination at the really end.

So 
```
$ uhb_whitelist -f xxx -o xxx
```

is okey.

Reply:
I wrote to a seperate temp file since the whitelist tool ands a blank line between every line. The temp file allows me to use grep to remove blank lines without input and output files being the same

Reply:
OK, I have changed it to be inplace, and added -m -p 60